dev-docs/Cloud-Env-Upgrade.md

```diff
@@ -0,0 +1,54 @@
+# Cloud Environment Upgrade Testing
+
+The [`Test Upgrade Environment`](https://github.com/elastic/cloudbeat/actions/workflows/upgrade-environment.yml) GitHub action automates the process of deploying a fully-featured cloud environment, pre-configured with all integrations (KSPM, CSPM, and D4C).
+It also facilitates the upgrade of the environment to a new version of the ELK stack and all installed agents, along with performing findings retrieval checks.
```

Comment:
Could you please provide more information on the specific version range for the upgrade test?

Reply:
Documentation is updated